zigzag/data/configs/zigzag-config-example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think we should have documentation on what all the possible config options are.